### PR TITLE
Commit on the Branch

### PR DESCRIPTION
**Pull request by Siddhant111**
Branches can be Merged

### CODE REVIEW

**Comment by Sid130** [comment]
This has no Conflicts and can be reviewed

**Review by Siddhant111 — COMMENTED** [review]
Changed First to Second

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
 
-    <h1>First Commit Buddy</h1>
```
Comment (Sid130) [inline index.html:11]: Changed First to Second to see the changes

Comment (testsyren) [inline index.html:11]: great stuff

**Review by testsyren — APPROVED** [review]

**Pull request merged**